src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/FancyZonesViewModel.cs

```diff
@@ -507,7 +529,7 @@ private void LaunchEditor()
             SendConfigMSG("{\"action\":{\"FancyZones\":{\"action_name\":\"ToggledFZEditor\", \"value\":\"\"}}}");
         }
 
-        public void RaisePropertyChanged([CallerMemberName] string propertyName = null)
+        public void NotifyPropertyChanged([CallerMemberName] string propertyName = null)
```

Comment:
For some context on this rename: the previous name **RaisePropertyChanged** was raising [CA1030](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1030?view=vs-2019). However it looks like we are already using an event-based pattern and that this method is a helper function that raises the actual event, so I've just renamed it to suppress the warnings.